src/ansys/fluent/core/workflow.py

```diff
@@ -1256,8 +1193,23 @@ def last_child(self) -> BaseTask:
         """
         children = self.tasks()
         if children:
+            py_name = self._get_python_names_for_compound_child()
```

Comment:
Does `_get_python_names_for_compound_child` only work for the last child of a compound task?

Reply:
@raph-luc, yes as this step always follows the insertion of a Compound child task.